scripts/generate-PR-body.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+function title {
+    echo "Sync with ethereum/solidity@develop $(date +%d-%m-%Y)"
```

Comment:
Can you include the tag in PR title (and commit description)?

On the call we agreed to leave labels for #15 but the title change is simpler so it was meant to be still applied here.

Reply:
will this be ok?
```
"Sync with ethereum/solidity@$(git describe --tags --always english/develop) $(date +%d-%m-%Y)"
```

Reply:
After the tests, I think putting the most recent tag in PR title can be misleading. The source of the PR is always the most recent commit, the most recent tag is not always the most recent revision.

Reply:
Oh, you mean that the last tag is e.g. 0.8.13 while the commits now will be a part of 0.8.14? That's true, but getting that is (slightly) more complicated. You can get that by running [scripts/get_version.sh](https://github.com/ethereum/solidity/blob/develop/scripts/get_version.sh).

But we still need some way to separate PRs for one version from another. If you can run that script to get the actual version, great. If not, I think that the version from the tag is better than nothing.